### ISSUE TITLE
it can not work

### ISSUE THREAD

**Issue by xhrwp** [knsv/mermaid#167]
![qq](https://cloud.githubusercontent.com/assets/10394828/8018668/9d14f43e-0c5a-11e5-9c62-fc501c9c6105.png)

**Comment by knsv** [comment]
I have added an example in test/web_render.html. The editor is not yet updated

**Comment by xhrwp** [comment]
thank you for your help

**Comment by xhrwp** [comment]
but i found another problem ，the file d3.js is not in the github。
![20150607121123](https://cloud.githubusercontent.com/assets/10394828/8022603/e636036a-0d0e-11e5-88b3-b2cba8355453.png)

**Comment by knsv** [comment]
Sorry about that. I am testing the bundling of mermaid in that branch.

If you instead of using mermaid.slim.js switch to mermaid.js that problem will be resolved.  

--  
Knut Sveidqvist
Sent with Sparrow (http://www.sparrowmailapp.com/?sig)

**Comment by xhrwp** [comment]
thank you! i have sollved it。i have a  question。 will  you add the export image function in the future?

**Comment by knsv** [comment]
with the render function you will get the sag code and can then save it as in the meta code below

```
var svgCode = mermaidAPI.render(’id1’,’graph TB\na—>b’);

saveFile(’myFile.svg’,svgCode);
```

This will be released in version 0.5.0  

--  
Knut Sveidqvist
Sent with Sparrow (http://www.sparrowmailapp.com/?sig)

**Comment by knsv** [comment]
I think is resolved. If you do not reopen the issue or create a new issue.
